i3pystatus/net_speed.py

```diff
@@ -12,23 +12,56 @@
 class NetSpeed(IntervalModule):
     """
     Attempts to provide an estimation of internet speeds.
-    Requires: speedtest_cli
+    Requires: speedtest-cli/modularize-2
```

Comment:
@drwahl Thanks, I havent been quite sure how to include this kind of dependency properly since the modularize-2 branch hasn't been merged yet.
And thanks for the conf thing, I was wondering if there was some other place I forgot to check.